ckanext/ontario_theme/plugin.py

```diff
@@ -181,12 +181,6 @@ def resource_display_name(resource_dict):
 
 ckan.lib.helpers.resource_display_name = resource_display_name
 
-def help():
-    '''New help page for site.
-    '''
-    return render_template('home/help.html')
-
-
 def csv_dump():
```

Comment:
Can you add an additional new line above line 184? So there are 2 new lines before `def csv_dump():`